spec/unit/facter/firewalld_version_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe 'firewalld_version' do
+
+  before :each do
+    Facter.clear
+    Facter::Core::Execution.stubs(:exec).with('uname -s').returns('Linux')
+    Facter::Util::Resolution.stubs(:which).with('firewall-cmd').returns('/usr/bin/firewall-cmd')
```

Comment:
Is this stub correct?  eg should it be `Facter::Core::Execution.stubs(:which)`??

Reply:
@alexjfisher Actually, the code was wrong and the test was right :-|